games-action/postal/postal-1.0.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="An isometric shooter game"
+DESCRIPTION="Postal is a port based on the open sourced original game code
+
+Please note you will need the original game files to play this game.
+
+To play: copy the Linux data folders (res and title) to: 
+/boot/home/config/non-packaged/data/PostalPlus
+"
+HOMEPAGE="https://runningwithscissors.com/games/postal/"
+COPYRIGHT="1997-2003 Running With Scissors"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/classiccoding/postal-1-open-source/archive/refs/heads/master.zip"
+CHECKSUM_SHA256="871f0cd25c278ee5bd4aaaaa863914f53ae238f4fb8ff0cdd4f2ca6d0fbfbeae"
+SOURCE_DIR="postal-1-open-source-master"
+PATCHES="postal-$portVersion.patchset"
+ADDITIONAL_FILES="postal.rdef"
+
+GLOBAL_WRITABLE_FILES="
+	settings/POSTAL.INI keep-old
+	"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	postal$secondaryArchSuffix = $portVersion
+	app:postal = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make HAIKU=1 $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Postal
+	cp bin/postal1 $appsDir/Postal/postal
```

Comment:
If you don't intend to install more than a binary, it can be located in $appsDir.